src/pages/api/dokumenter/[journalpostId]/[dokumentId]/forhandsvisning.ts

```diff
@@ -2,9 +2,16 @@ import { NextApiHandler } from "next";
 import { v4 as uuidv4 } from "uuid";
 import { withSentry } from "@sentry/nextjs";
 import { getSession } from "../../../../../lib/auth.utils";
+import { Stream } from "stream";
 
 const audience = `${process.env.SAF_SELVBETJENING_CLUSTER}:teamdokumenthandtering:${process.env.SAF_SELVBETJENING_SCOPE}`;
 
+export const config = {
```

Comment:
slipper å få API 

`response for /api/dokumenter/598126548/624874530/forhandsvisning exceeds 4MB. API Routes are meant to respond quickly. https://nextjs.org/docs/messages/api-routes-response-size-limit`

men lurer på sette en eksplisitt grense